interfaces/Portal/src/app/services/program-phase.service.ts

```diff
@@ -14,8 +14,8 @@ export class Phase {
 @Injectable({
   providedIn: 'root',
 })
-export class ProgramPhaseService {
-  public activePhaseName: ProgramPhase;
+export class ProgramTabService {
```

Comment:
Is it worth it to also rename all these internal "phase"-things to "tab"-things?

Or would it be similar effort to "refactor this thing away"?

Reply:
When we work on the new Payment-flow (and remove the current Payment-page), this whole service will not be necessary anymore.
So I think we can live with it for a little bit longer and not rename-all-the-things for now...